guides/user/pipeline/managing-pipelines/index.md

```diff
@@ -0,0 +1,135 @@
+3. Choose **Pipeline** from the drop down menu and name your pipeline.
+
+After you create your pipeline, add stages to specify the actions that your
+pipeline will perform.
+
+### Add a stage
+
+1. Select **Add stage** from your pipeline configuration screen.
+  ![](images/add-stage.png)
+2. Set the stage type using the drop-down menu.
+3. If this is not the first stage in your pipeline, make sure that this stage
+depends on the desired upstream stage(s) using the **Depends on** field.
+  ![](images/stage-depends-on.png)
+
+You can add as many stages as your pipeline needs, in any order that makes sense
+for you.
+
+### Add a trigger
+
+Make sure that you are editing the **Configuration** stage of your pipeline.
+
+![](images/configuration-stage.png)
+
+1. Select **Add trigger**.
+  ![](images/add-trigger.png)
+2. Choose your desired trigger type from the drop-down menu that appears, and
+input any further required configuration.
+
+For further information on how triggers work, see the documentation on pipeline
+triggers.
+<!-- TODO(nhayes): link here to pipeline triggers overview once it exists. -->
+
+## Disable a pipeline
+
+Disabling a pipeline prevents any triggers from firing, as well as preventing
+users from running it manually.
+
+1. From the pipelines tab, click **Configure** to modify an existing pipeline.
+  ![](images/select-configure.png)
+2. Click **Pipeline actions** in the upper right corner, and select **Disable**.
+  ![](images/pipeline-actions.png)
+
+In order to re-enable your disabled pipeline, select **Pipeline actions** and
+choose **Enable**.
+
+## Delete a pipeline
+
+1. From the pipelines tab, click **Configure** to modify an existing pipeline.
+  ![](images/select-configure.png)
+2. Click **Pipeline actions** in the upper right corner, and select **Delete**.
+  ![](images/pipeline-actions.png)
+
+## Edit as JSON
+
+First, some background: Spinnaker represents pipelines as JSON behind the
+scenes. Any changes you make to your pipeline using the UI are converted to JSON
+when Spinnaker saves the pipeline.
+
+When you use the **Edit as JSON** feature, you are directly editing the payload.
+This can be useful for working around limitations of the UI. However, the JSON
+you write here *is not validated* -- **Edit as JSON** essentially allows you to
+modify the pipeline via a free-form textbox. Be careful! It is very easy to
+break the pipeline.
+
+In order to edit your pipeline as JSON:
+
+1. From the pipelines tab, click **Configure** to modify an existing pipeline.
+  ![](images/select-configure.png)
+2. Click **Pipeline actions** in the upper right corner, and select
+**Edit as JSON**.
+  ![](images/pipeline-actions.png)
+
+## View and restore revision history
+
+Each time you save your pipeline, the current version is added to revision
+history. You can use revision history to diff two versions of a pipeline or to
+restore an older version of a pipeline.
+
+> *Note*: minio and redis do not support version history.
```

Comment:
Maybe can clarify that minio and redis are backing stores for the pipeline repository, and maybe provide a link to another section where you can read more about front50 storage?

Reply:
I updated this to clarify that minio/redis are used to store configuration files, but couldn't find documentation that explains much about front50 storage. Maybe I just missed it though - did you have anything in particular in mind? 

I didn't want to go too in-depth about front50 here since it's only tangentially related, but let me know if you think I should add more detail or mention it more explicitly.

Reply:
It SGTM as is.

Reply:
@jtk54 
@lwander 
If Redis and Minio don't support version history, but we have the versioning functionality, how does that work? It's handled by Front50? If so, it might be worthwhile, @nhayes , to mention something here. But only if it matters to the user. In fact, @lwander , why does the user need to know that Redis and Minio don't support versioning?